source/MapDetailPanel.cpp

```diff
@@ -589,6 +614,31 @@ void MapDetailPanel::DrawInfo()
 		uiPoint.Y() += 20.;
 	}
 
+	autoBuyY = uiPoint.Y();
+
+	// "Buy best" button goes after trade prices
+	if(player.CanTrade())
+	{
+		int canBuyBest = player.HasBestTrade(selectedSystem);
+		if(canBuyBest || player.Cargo().CommoditiesSize())
+		{
+			const Sprite *buyAllSprite = SpriteSet::Get("ui/planet dialog button"); //160x60
+			const auto alignCenter = Layout(160, Alignment::CENTER);
+			SpriteShader::Draw(buyAllSprite, uiPoint + Point(70, 35));
+			if(canBuyBest == 0)
+			{
+				font.Draw({"Sell Commodi_ties", alignCenter}, uiPoint + Point(-10, 28), medium);
+			}
+			else
+			{
+				font.Draw({canBuyBest == 2 ? "Fill Ship with" : "Fill Fleet with", alignCenter}, uiPoint + Point(-10, 20), medium);
+				font.Draw({"_Best _Trade", alignCenter}, uiPoint + Point(-10, 37), medium);
```

Comment:
```suggestion
				font.Draw({"_Best Trade", alignCenter}, uiPoint + Point(-10, 37), medium);
```
Why is there an underscore before the T? it isn't a valid shortcut

Reply:
That one actually uses two shortcuts, 'b' and 't', one meaning 'buy' and one for 'trade' - 'buy' will only buy but 'trade' will buy or sell as the button changes. Similar to the outfit panel 'b'/'i' buying or installing, but doing the same thing in certain contexts.

('s' was already used on that screen for 'shipyard' so I came upon this alternate 't')

Reply:
Oh okay I got it, but do add a comment somewhere saying that if there isn't already one